lib/Remote/RemoteWebElement.php

```diff
@@ -477,6 +485,39 @@ public function equals(WebDriverElement $other)
         ]);
     }
 
+    /**
+     * Attempt to click on a child level element.
+     *
+     * This provides a workaround for geckodriver bug 653 whereby a link whose first element is a block-level element
```

Comment:
Also rather full URL (maybe both github + bugzilla here) then just bug ID.

Reply:
I've added a @see link for both